lib/rtcpeerconnection/firefox.js

```diff
@@ -110,93 +103,10 @@ function FirefoxRTCPeerConnection(configuration) {
   });
 
   util.proxyProperties(PeerConnection.prototype, this, peerConnection);
-
-  // NOTE(mroberts): We use the adapter.js workaround for providing track events.
-  if (!('ontrack' in PeerConnection.prototype)) {
-    peerConnection.addEventListener('addstream', function onaddstream(addStreamEvent) {
-      var mediaStream = addStreamEvent.stream;
-
-      // NOTE(mmalavalli): We are not using MediaStream#onaddtrack event listeners
-      // for shimming PeerConnection#ontrack like we do for Chrome because it
-      // is not yet supported in Firefox (support is slated for Firefox 50).
-      // That's why we don't see this event being used in adapter.js's Firefox
-      // shim.
-      // Reference: https://developer.mozilla.org/en-US/docs/Web/API/MediaStream#Browser_compatibility
-
-      mediaStream.getTracks().forEach(function(mediaStreamTrack) {
-        var newEvent = new Event('track');
-        newEvent.track = mediaStreamTrack;
-        newEvent.streams = [mediaStream];
-        self.dispatchEvent(newEvent);
-      });
-    });
-  }
```

Comment:
What's the RTCTrackEvent support story for Firefox? e.g., what is ESR supporting? What are recent Ubuntus shipping? etc.

Reply:
Per above, we can remove.